src/components/CivicSubscriptionView/CivicSubscriptionView.vue

```diff
@@ -602,7 +602,7 @@ export default {
           !this.isSelf &&
           !this.getUserInfoById(this.authorId)
         ) {
-          await this.fetchUserInfo(this.authorId);
+          await this.fetchUserInfo(this.authorId, { types: ['creator'] });
```

Comment:
Should this be `({ id: this.authorId, types: ['creator'] })`?

Reply:
Fixed